frameworks/kafka/tests/test_ssl_kerberos_custom_domain_auth.py

```diff
@@ -0,0 +1,204 @@
+import logging
+import uuid
+import pytest
+
+import sdk_auth
+import sdk_cmd
+import sdk_hosts
+import sdk_install
+import sdk_marathon
+import sdk_utils
+
+
+from security import transport_encryption
+
+
+from tests import auth
+from tests import config
+from tests import test_utils
+
+
+log = logging.getLogger(__name__)
+
+
+pytestmark = pytest.mark.skipif(sdk_utils.is_open_dcos(),
```

Comment:
Aslo `1.10+` right? We need secrets for this and 1.9 doesn't have them?

Reply:
The individual tests are annotated for some reason. I wasn't a good citizen and didn't pull it up to the top level.

Reply:
Yes, the individual annotations are also ok. I've just been bitten before by forgetting to add the annotation. No need to change it though.